### PR TITLE
add hardware report tile

### PR DESCRIPTION
**Pull request by potch**
* Added Firefox Hardware Report tile
* Removed Heroes of Paragon Tile
* Swapped openwebgames with Web Assembly to keep wasm on top row

### CODE REVIEW

File: public_html/index.html [hunk 1]
```diff
@@ -35,28 +35,39 @@
...
             </a>
           </li>
           <li class="highlight__item">
+            <a href="http://openwebgames.com" class="link">
+              <div class="imageWrapper">
+                <img src="assets/img/tile-owg.jpg">
+              </div>
+              <figcaption>
+                <h2 class="highlight__item__title">Open Web Games Test Suite</h2>
+                <p>Test your browser online gaming capability</p>
```
Comment (cvan) [inline public_html/index.html]: maybe reword to `Test your browser's support for online games`?

File: public_html/index.html [hunk 2]
```diff
@@ -79,17 +90,6 @@ <h2 class="highlight__item__title">MozVR</h2>
             </a>
           </li>
           <li class="highlight__item">
-            <a href="https://medium.com/@vvergon/webgl-matures-into-a-commercial-ready-technology-ba71e254cae6#.bju7pnc1r" class="link">
-              <div class="imageWrapper">
-                <img src="assets/img/tile-heroes.jpg">
```
Comment (cvan) [inline public_html/index.html]: want to `git rm` this image?

File: public_html/index.html [hunk 3]
```diff
@@ -35,28 +35,39 @@
         <h4 class="section-title">Top Stories</h4>
         <ul>
           <li class="highlight__item">
-            <a href="http://openwebgames.com" class="link">
+            <a href="https://metrics.mozilla.com/firefox-hardware-report" class="link">
```
Comment (cvan) [inline public_html/index.html]: want to add the trailing slash (to save a redirect)?

File: public_html/index.html [hunk 4]
```diff
@@ -35,28 +35,39 @@
...
               <div class="imageWrapper">
-                <img src="assets/img/tile-owg.jpg">
+                <img src="assets/img/tile-hardwarereport.jpg">
```
Comment (cvan) [inline public_html/index.html]: I would remove the graphical text from this tile image - so it can be easily localised (and styled appropriately responsively). (I had a L10n system in place here for the original GDC site, but it was removed during the redesign of this site.)

File: public_html/index.html (hunk 1 above)
Comment (potch) [inline public_html/index.html]: This was the prior copy, I just moved this in the document.

File: public_html/index.html (hunk 4 above)
Comment (potch) [inline public_html/index.html]: Asset was provided by someone else, just sticking with the current convention (see webassembly image)

File: public_html/index.html (hunk 2 above)
Comment (potch) [inline public_html/index.html]: done.

File: public_html/index.html (hunk 1 above)
Comment (cvan) [inline public_html/index.html]: ah, okay - but that reword sound good to you?

File: public_html/index.html (hunk 2 above)
Comment (cvan) [inline public_html/index.html]: I don't see it

File: public_html/index.html [hunk 5]
```diff
@@ -35,28 +35,39 @@
         <h4 class="section-title">Top Stories</h4>
         <ul>
           <li class="highlight__item">
-            <a href="http://openwebgames.com" class="link">
+            <a href="https://metrics.mozilla.com/firefox-hardware-report/" class="link">
               <div class="imageWrapper">
-                <img src="assets/img/tile-owg.jpg">
+                <img src="assets/img/tile-hardwarereport.jpg">
```
Comment (cvan) [inline public_html/index.html]: it seems you forgot to include this when you rebased

Comment (cvan) [inline public_html/index.html]: also, per my earlier comment, we sure we want to include that English text hardcoded in the graphic? IMO, we could render it as regular text styled with CSS (and can be later localised).

File: public_html/index.html (hunk 2 above)
Comment (potch) [inline public_html/index.html]: borked the push I think

**Comment by potch** [comment]
somehow supremely horked this with some force pushes. gonna close and try again.
